conan.cmake

```
@@ -415,6 +415,13 @@ function(conan_cmake_install)
     else()
       set(CONANFILE ".")
     endif()
+    if(ARGUMENTS_LOCKFILE)
+      if(IS_ABSOLUTE ${ARGUMENTS_LOCKFILE})
```

I am thinking something like this to inform the user (with a nicer message)

```suggestion
    if(ARGUMENTS_LOCKFILE)
      if(ARGUMENTS_PROFILE OR ARGUMENTS_OPTIONS OR ARGUMENTS_SETTINGS)
        message(FATAL_ERROR "Cannot mix LOCKFILE with other graph changing arguements")
      endif()
      if(IS_ABSOLUTE ${ARGUMENTS_LOCKFILE})
```

---

- what about, maybe also, adding a if branch on line (currently) 444 , `set(conan_args install` , and just ignoring all args that make no sense when a lockfile is used ?